### PR TITLE
PS-720_alpha-tiff-center

### PR DESCRIPTION
**Pull request by jygaulier**
paste layers one by one, centered
### BC-Break
- options `size` and `position` are removed
- option `transparency` is renamed `opacity` to conform to values 0=transparent ; 100=full opaque color

### CODE REVIEW

**Review by 4rthem — CHANGES_REQUESTED** [review]
You should introduce a new Filter, not overriding the Liip's one because it will:
- break BC as you mentioned
- add confusion between documentations
Name suggestion: `BackgroundFillFilter`?

**Comment by jygaulier** [comment]
nb: liip filter is *not* overwritten (it stays buggy...). we just use our own filter (namespace) with the same name (so no need to change conf).

File: lib/php/rendition-factory/src/Transformer/Image/Imagine/BackgroundFilterLoader.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    public function load(ImageInterface $image, array $options = [])
+    {
+        $background = $image->palette()->color(
+            $options['color'] ?? '#fff',
+            $options['opacity'] ?? 100,
+        );
+        $canvas = $this->imagine->create($image->getSize(), $background);
+
+        // DO NOT REMOVE THIS LINE
```
Comment (4rthem) [inline lib/php/rendition-factory/src/Transformer/Image/Imagine/BackgroundFilterLoader.php:24]:
```suggestion
```

**Review by 4rthem — CHANGES_REQUESTED** [review]
I maintain the idea you should find a different name for your filter to avoid confusion with the LiipImage document which we rely on today.

**Review by 4rthem — APPROVED** [review]

**Pull request merged**